docs/plotting.rst

```diff
@@ -41,9 +41,9 @@ We train on a 150 random points in :math:`[0, 1]`.
 
 >>> import numpy
 >>> from fuel.streams import DataStream
->>> from fuel.datasets import ContainerDataset
+>>> from fuel.datasets import IterableDataset
 >>> sample = theano.tensor.scalar('data')
->>> data_stream = DataStream(ContainerDataset(
+>>> data_stream = DataStream(IterableDataset(
```

Comment:
Mm, I see your point, but I hate the idea of throwing the name around for this one case and using "DatasetOfIterables" or something weird. "Iterables" are the only correct name IMHO, because the only property it relies on is being able to iterate over things. `ContainerDataset` doesn't work, because there is no requirement for any of the sources to be containers.

Any  suggestions? I'd be fine with keeping `IterableDataset` though; it should be clear from reading the summary line alone what it is already.